doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -768,7 +768,7 @@ zcbor
 Trusted Firmware-M
 ==================
 
-|no_changes_yet_note|
+* Added possibility to read UICR.OTP registers through platform services.
```

Comment:
This doesn't seem to cover the same ground as the added documentation. Is the read service something that was already there, and it is now documented to show how to read the OTP registers from the UICR?